infra/cifuzz/actions/build_fuzzers/Dockerfile

```diff
@@ -14,8 +14,28 @@
 #
 ################################################################################
 # Docker image to run CIFuzz in.
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -y git \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gnupg2 \
+    software-properties-common \
+    python3
+
+
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - && apt-key fingerprint 0EBFCD88
+RUN add-apt-repository \
+   "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+   xenial \
+   stable"
+
+RUN apt-get update && apt-get install docker-ce docker-ce-cli containerd.io -y
+
+ENV OSS_FUZZ_ROOT=/opt/oss-fuzz
+RUN git clone https://github.com/evverx/oss-fuzz.git ${OSS_FUZZ_ROOT}
 
-FROM gcr.io/oss-fuzz-base/cifuzz-base
```

Comment:
FWIW I'll remove the second commit (where I pointed CIFuzz to my repository) later but this base image makes it kind of hard to test CIFuzz "locally". It took me about half an hour to figure out why it kept failing with a TypeError saying that `build_fuzzers` didn't expect the "sanitizer" keyword.

Reply:
Yeah I've run into this problem a few times myself. One of my TODOs is to document developing CIFuzz.